### ISSUE TITLE
unnecessary_pragma, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#2730]
```abap
FORM entry USING
    return_code TYPE i
    us_screen   TYPE abap_bool ##NEEDED.
```
